app/scripts/HiGlassComponent.js

```diff
@@ -3088,6 +3116,23 @@ class HiGlassComponent extends React.Component {
           }
         }
 
+        if (
```

Comment:
You have a nice comment in the block above stating that it's executed if there's no `fromView`. Maybe another comment here would be helpful to refresh the readers memory that this block is entered if `fromView` is present?